scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -45,15 +49,63 @@ public static BuildTargetResult InitPublish(BuildTargetContext c)
         [Environment("PUBLISH_TO_AZURE_BLOB", "1", "true")] // This is set by CI systems
         public static BuildTargetResult Publish(BuildTargetContext c)
         {
+            if (ShouldPublishToLatest())
+            {
+                string targetContainer = $"{Channel}/Binaries/Latest/";
+
+                foreach (string blob in _latestBlobs)
+                    AzurePublisherTool.CopyBlob(blob.Replace("/dotnet/", ""), (targetContainer + Path.GetFileName(blob)).Replace(CliNuGetVersion, "latest").Replace(SharedFrameworkNugetVersion, "latest"));
+
+                PublishLatestCliVersionTextFile(c);
+                PublishLatestSharedFrameworkVersionTextFile(c);
+            }
+
             return c.Success();
         }
 
+        private static bool ShouldPublishToLatest()
+        {
+             Dictionary<string, bool> runtimes = new Dictionary<string, bool>()
+             {
+                 {"win7-x64", false },
+                 {"win7-x86", false },
+                 {"osx.10.10", false },
+                 {"rhel.7", false },
+                 {"ubuntu.14.04", false },
+             };
+
+            _latestBlobs = new List<string>(AzurePublisherTool.ListBlobs($"{Channel}/Binaries/{CliNuGetVersion}/"));
+
+            foreach (string file in _latestBlobs)
+            {
+                string name = Path.GetFileName(file);
+                string key = string.Empty;
+
+                foreach (string runtime in runtimes.Keys)
+                {
+                    if (name.StartsWith($"runtime.{runtime}"))
```

Comment:
per offline conversation, let's add a "FinalizeBuildPublishing" target that in turn calls the "PublishVersionBadge" Target. This will show in code that PublishVersion is a Finalization step.